dev/com.ibm.ws.jsp.2.3_fat/test-applications/TestServlet.war/resources/moreoutput.jsp

```diff
@@ -0,0 +1,19 @@
+<%--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+--%>
+<html>
+<body>
+<jsp:include page="include.jsp" flush="true"/><br>
+<p>more text following flush
+
+</body>
```

Comment:
Can you format this jsp a bit better?

Reply:
I put the <br> on its own line.  Not sure what else would improve format.